tensorflow/workspace2.bzl

```diff
@@ -184,9 +184,9 @@ def _tf_repositories():
         name = "mkl_dnn_acl_compatible",
         build_file = "//third_party/mkl_dnn:mkldnn_acl.BUILD",
         patch_file = ["//third_party/mkl_dnn:onednn_acl.patch"],
```

Comment:
Do you still need the patch file? I guess you do since we are updating from 2.6-rc to 2.6?

Reply:
Yes, for now it's still needed.
This patch was introduced with #55404 as part of the fix for accuracy issues seen in training workloads - it should be possible to remove the patch for a future oneDNN release, but at present the fix is needed for both the RC and Release versions of oneDNN.